src/lib.rs

```diff
@@ -9,6 +7,7 @@ pub use self::signature::*;
 use parity_wasm::elements::*;
 
 pub const DEFAULT_SYMBOL_NAME: &str = "___SIGNATURE";
+pub const DEFAULT_CUSTOM_SECTION_NAME: &str = "signature_wasmsign";
```

Comment:
Question: Perhaps we should reuse the "signature" Custom Section and format from https://github.com/frehberg/wasm-sign? Obiviously, with a different signature type.